### ISSUE TITLE
hIncrByFloat command have no effect

### ISSUE THREAD

**Issue by GerBawn** [phpredis/phpredis#995]
```
$redis = new RedisCluster(null, ['127.0.01:6379', '127.0.0.1:7379', '127.0.0.1:8379']);
var_dump($redis->hIncrByFloat('test_incr', 'key1', 0.1));
var_dump($redis->hgetall('test_incr'));
```
After run these code, will output:
```
float(0)
array(1) {
  ["key1"]=>
  string(1) "0"
}
```
No matter how many times running, The output will always like above. But other command like `hIncrBy`, `hSet` can work correctly.

my environment:
php-5.6.27
phpredis-2.2.8
centos7
redis-3.2.5

**Comment by yatsukhnenko** [comment]
@GerBawn, could you try code from `develop` brunch?

**Comment by GerBawn** [comment]
This problem occurs on the production machine. In development machine, it can work correctly.

**Comment by yatsukhnenko** [comment]
@GerBawn, what the difference between your development and production machines?

**Comment by yatsukhnenko** [comment]
@GerBawn, this issue should be fixed. Test please.

**Comment by yatsukhnenko** [comment]
Probably fixed in 3.1.1RC
